# require config from /etc/webapps
# chdir .. before including common.php

--- phorum-5.2.14/common.php~	2010-01-06 19:13:05.000000000 +0200
+++ phorum-5.2.14/common.php	2010-01-06 19:15:15.204911869 +0200
@@ -84,22 +84,22 @@
     ob_start();
 
     // Load configuration.
-    if (! include_once( "./include/db/config.php" )) {
+    if (! include_once( "/etc/webapps/phorum/config.php" )) {
         print '<html><head><title>Phorum error</title></head><body>';
         print '<h2>Phorum database configuration error</h2>';
 
         // No database configuration found.
-        if (!file_exists("./include/db/config.php")) { ?>
+        if (!file_exists("/etc/webapps/phorum/config.php")) { ?>
             Phorum has been installed on this server, but the configuration<br/>
             for the database connection has not yet been made. Please read<br/>
             <a href="docs/install.txt">docs/install.txt</a> for installation
             instructions. <?php
         } else {
-            $fp = fopen("./include/db/config.php", "r");
+            $fp = fopen("/etc/webapps/phorum/config.php", "r");
             // Unable to read the configuration file.
             if (!$fp) { ?>
                 A database configuration file was found in
-                ./include/db/config.php,<br/>but Phorum was unable to read it.
+                config.php,<br/>but Phorum was unable to read it.
                 Please check the file permissions<br/>for this file. <?php
             // Unknown error.
             } else {
--- phorum-5.2.14/include/admin/install.php~	2010-01-06 19:23:20.000000000 +0200
+++ phorum-5.2.14/include/admin/install.php	2010-01-06 19:23:35.421527751 +0200
@@ -25,7 +25,7 @@
 include_once("./include/api/user.php");
 
 if(!phorum_db_check_connection()){
-    echo "A database connection could not be established.  Please edit include/db/config.php.";
+    echo "A database connection could not be established.  Please edit config.php.";
     return;
 }
 
--- phorum-5.0.21/htdocs/admin.php	2005-03-17 19:41:22.000000000 +0200
+++ phorum-5.0.21.pld/htdocs/admin.php	2006-03-08 18:23:04.000000000 +0200
@@ -26,6 +26,7 @@
     // the code run faster.
     error_reporting  (E_ERROR | E_WARNING | E_PARSE);
 
+    chdir("..");
     include_once "./common.php";
     include_once "./include/users.php";
 
--- phorum-5.0.21/htdocs/control.php	2005-10-29 00:02:51.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/control.php	2006-03-08 18:23:04.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','control');
 
+chdir("..");
 include_once("./common.php");
 
 phorum_require_login();
--- phorum-5.0.21/htdocs/file.php	2005-03-17 19:41:22.000000000 +0200
+++ phorum-5.0.21.pld/htdocs/file.php	2006-03-08 18:23:04.000000000 +0200
@@ -23,6 +23,7 @@
 ini_set ( "zlib.output_compression", "0");
 ini_set ( "output_handler", "");
 
+chdir("..");
 include_once("./common.php");
 
 // checking read-permissions
--- phorum-5.0.21/htdocs/follow.php	2005-03-17 19:41:22.000000000 +0200
+++ phorum-5.0.21.pld/htdocs/follow.php	2006-03-08 18:23:04.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','subscribe');
 
+chdir("..");
 include_once("./common.php");
 
 phorum_require_login();
--- phorum-5.0.21/htdocs/index.php	2005-03-17 19:41:22.000000000 +0200
+++ phorum-5.0.21.pld/htdocs/index.php	2006-03-08 18:23:04.000000000 +0200
@@ -17,6 +17,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 define('phorum_page','index');
 
+chdir("..");
 include_once( "./common.php" );
 
 include_once( "./include/format_functions.php" );
--- phorum-5.0.21/htdocs/list.php	2005-04-01 20:10:41.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/list.php	2006-03-08 18:23:04.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','list');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/format_functions.php");
 
--- phorum-5.0.21/htdocs/login.php	2005-10-15 13:51:01.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/login.php	2006-03-08 18:23:04.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','login');
 
+chdir("..");
 include_once( "./common.php" );
 include_once( "./include/users.php" );
 include_once( "./include/email_functions.php" );
--- phorum-5.2.14/htdocs/moderation.php~	2010-01-06 19:21:12.000000000 +0200
+++ phorum-5.2.14/htdocs/moderation.php	2010-01-06 19:22:37.991532354 +0200
@@ -19,6 +19,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 define('phorum_page','moderation');
+chdir("..");
 include_once("./common.php");
 include_once("./include/moderation_functions.php");
 include_once("./include/thread_info.php");
--- phorum-5.1.25/htdocs/posting.php~	2007-06-29 18:03:25.000000000 +0300
+++ phorum-5.1.25/htdocs/posting.php	2007-12-27 18:49:04.318221622 +0200
@@ -55,6 +55,7 @@
     define('phorum_page', 'post');
 }
 
+chdir("..");
 include_once("./common.php");
 include_once("include/format_functions.php");
 
--- phorum-5.0.21/htdocs/profile.php	2005-04-15 16:37:05.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/profile.php	2006-03-08 18:23:05.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','profile');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/email_functions.php");
 include_once("./include/format_functions.php");
--- phorum-5.0.21/htdocs/read.php	2005-04-01 20:10:41.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/read.php	2006-03-08 18:23:05.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','read');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/email_functions.php");
 include_once("./include/format_functions.php");
--- phorum-5.0.21/htdocs/register.php	2005-10-14 23:46:40.000000000 +0300
+++ phorum-5.0.21.pld/htdocs/register.php	2006-03-08 18:23:05.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','register');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/users.php");
 include_once("./include/profile_functions.php");
--- phorum-5.0.21/htdocs/search.php	2005-11-03 23:56:46.000000000 +0200
+++ phorum-5.0.21.pld/htdocs/search.php	2006-03-08 18:23:05.000000000 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','search');
 
+chdir("..");
 include_once("./common.php");
 
 if(!phorum_check_read_common()) {
--- phorum/include/admin/header.php~	2009-07-22 14:58:57.000000000 +0300
+++ phorum/include/admin/header.php	2010-01-06 20:02:05.938378394 +0200
@@ -36,7 +36,7 @@
 
     // set the path to the CSS file to pull in
     $default_admin_css_file = 'default.css';
-    $admin_css_path = $PHORUM["http_path"].'/include/admin/css/' . $default_admin_css_file;
+    $admin_css_path = $PHORUM["http_path"].'/admin/css/' . $default_admin_css_file;
 
     /**
      * [hook]
--- phorum/htdocs/admin/css/default.css~	2010-01-06 20:13:13.000000000 +0200
+++ phorum/htdocs/admin/css/default.css	2010-01-06 20:13:19.085201493 +0200
@@ -118,19 +118,19 @@
 .icon-folder-up {
     width: 22px;
     padding-left: 22px;
-    background: url(../../../images/folder_up.png) 0 1px no-repeat;
+    background: url(../../images/folder_up.png) 0 1px no-repeat;
 }
 
 .icon-folder {
     width: 22px;
     padding-left: 22px;
-    background: url(../../../images/folder.png) 0 1px no-repeat;
+    background: url(../../images/folder.png) 0 1px no-repeat;
 }
 
 .icon-forum {
     width: 22px;
     padding-left: 22px;
-    background: url(../../../images/forum.png) 0 1px no-repeat;
+    background: url(../../images/forum.png) 0 1px no-repeat;
 }
 
 .PhorumAdminTableHead
@@ -154,7 +154,7 @@
 
 .PhorumAdminError
 {
-    background-image: url("../../../images/alert.gif");
+    background-image: url("../../images/alert.gif");
     background-position: 5px 5px;
     background-repeat: no-repeat;
     font-family: Lucida Grande, Lucida Sans Unicode, Verdana, Arial, Helvetica;
--- phorum-5.2.14/htdocs/addon.php	2010-01-11 16:01:43.649536786 +0200
+++ phorum-5.2.14-paths/htdocs/addon.php	2010-01-11 16:00:14.376410817 +0200
@@ -77,6 +77,7 @@
 
 define('phorum_page','addon');
 
+chdir("..");
 include_once( "./common.php" );
 
 // Bail out early if there are no modules enabled that implement
--- phorum-5.2.14/htdocs/ajax.php	2010-01-11 16:01:43.649536786 +0200
+++ phorum-5.2.14-paths/htdocs/ajax.php	2010-01-11 16:00:22.517084899 +0200
@@ -24,6 +24,7 @@
 
 define('phorum_page', 'ajax');
 
+chdir("..");
 require_once('./common.php');
 require_once('./include/api/json.php');
 
--- phorum-5.2.14/htdocs/changes.php	2010-01-11 16:01:44.246269735 +0200
+++ phorum-5.2.14-paths/htdocs/changes.php	2010-01-11 16:00:26.976419856 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','changes');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/format_functions.php");
 include_once("./include/diff_patch.php");
--- phorum-5.2.14/htdocs/css.php	2010-01-11 16:01:43.649536786 +0200
+++ phorum-5.2.14-paths/htdocs/css.php	2010-01-11 16:00:34.303030241 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 define('phorum_page','css');
+chdir("..");
 require_once('./common.php');
 
 // Set to FALSE to disable CSS compression.
--- phorum-5.2.14/htdocs/feed.php	2010-01-11 16:01:44.182870087 +0200
+++ phorum-5.2.14-paths/htdocs/feed.php	2010-01-11 16:00:39.736210271 +0200
@@ -20,6 +20,7 @@
 
 define("phorum_page", "feed");
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/format_functions.php");
 include_once("./include/feed_functions.php");
--- phorum-5.2.14/htdocs/javascript.php	2010-01-11 16:01:43.362870559 +0200
+++ phorum-5.2.14-paths/htdocs/javascript.php	2010-01-11 16:00:45.322876145 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 define('phorum_page','javascript');
+chdir("..");
 require_once("./common.php");
 
 // So we can use {URL->HTTP_PATH} in the templates.
--- phorum-5.2.14/htdocs/pm.php	2010-01-11 16:01:43.372872367 +0200
+++ phorum-5.2.14-paths/htdocs/pm.php	2010-01-11 16:00:55.022939080 +0200
@@ -32,6 +32,7 @@
 
 define('phorum_page','pm');
 
+chdir("..");
 include_once("./common.php");
 
 phorum_require_login();
--- phorum-5.2.14/htdocs/redirect.php	2010-01-11 16:01:44.182870087 +0200
+++ phorum-5.2.14-paths/htdocs/redirect.php	2010-01-11 16:01:06.719540737 +0200
@@ -24,6 +24,7 @@
 
 define('phorum_page', 'redirect');
 
+chdir("..");
 require_once("./common.php");
 
 if (isset($PHORUM["args"]["phorum_redirect_to"])) {
--- phorum-5.2.14/htdocs/report.php	2010-01-11 16:01:43.366204448 +0200
+++ phorum-5.2.14-paths/htdocs/report.php	2010-01-11 16:01:09.336206826 +0200
@@ -18,6 +18,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 define('phorum_page','report');
 
+chdir("..");
 include_once("./common.php");
 include_once("./include/email_functions.php");
 include_once("./include/format_functions.php");
--- phorum-5.2.14/htdocs/versioncheck.php	2010-01-11 16:01:44.253079929 +0200
+++ phorum-5.2.14-paths/htdocs/versioncheck.php	2010-01-11 16:01:11.172873461 +0200
@@ -24,6 +24,7 @@
 // so downtime of the phorum.org website won't affect the performance of
 // the admin interface for Phorum users.
 
+chdir("..");
 require_once('./common.php');
 
 if (isset($_COOKIE["phorum_upgrade_available"])) {
